test/namedCurry.js

```diff
@@ -0,0 +1,49 @@
+var R = require('../source');
+var eq = require('./shared/eq');
+
+describe('namedCurry', function() {
+  it('curries a single value', function() {
+    var f = R.namedCurry([['a'], ['b'], ['c'], ['d']], function({a, b, c, d}) {return (a + b * c) / d;}); // f(12, 3, 6, 2) == 15
```

Comment:
Note that theses comments here are wrong. It's a very minor thing, but that's no longer the correct signature of the functions..

We could change them to be more accurate, but I think it's just as good simply to dump them.

Reply:
Let's remove them.  They are no longer accurate, and they never were very helpful.